custom/ewsghana/tests/test_input_stock_view.py

```diff
@@ -192,6 +203,17 @@ def test_web_user_with_valid_parent_location_access(self):
         self.assertEqual(tsactive.products.count(), 2)
         self.assertEqual(len(list(formset)), tsactive.products.count())
 
+    def test_admin_web_user_access(self):
+        self.client.login(username=self.admin_username, password=self.admin_password)
+
+        view_url = reverse('input_stock', kwargs={'domain': TEST_DOMAIN, 'site_code': 'tsactive'})
+        response = self.client.get(view_url, follow=True)
+        self.assertEqual(response.status_code, 200)
+
+        view_url = reverse('input_stock', kwargs={'domain': TEST_DOMAIN, 'site_code': 'rsp'})
+        response = self.client.get(view_url, follow=True)
+        self.assertEqual(response.status_code, 200)
```

Comment:
I forgot to paste link to case: http://manage.dimagi.com/default.asp?190529#1068195.
Mike didn't write that now only admins should have access to input stock data so it seems existing logic should be leaved.